src/Sylius/Behat/Page/Shop/ProductReview/CreatePage.php

```diff
@@ -14,6 +14,7 @@
 namespace Sylius\Behat\Page\Shop\ProductReview;
 
 use FriendsOfBehat\PageObjectExtension\Page\SymfonyPage;
+use Sylius\Behat\Service\DriverHelper;
 use Webmozart\Assert\Assert;
 
 class CreatePage extends SymfonyPage implements CreatePageInterface
```

Comment:
You can extend `\Sylius\Behat\Page\Shop\Page` and use the `waitForElementUpdate` function.